packages/odyssey-storybook/src/guidelines/OdysseyTranslationProvider.stories.mdx

```diff
@@ -89,3 +89,13 @@ The following keys can have their values overriden:
 - `severity.info`
 - `severity.error`
 - `toast.close.text`
+
+## Passing the language from the Monolith
+
+```
+import { getOktaLocale } from '@okta/ui-libraries-monolith';
+
+<OdysseyProvider languageCode={getOktaLocale()}>
+        <CssBaseline>
+....
+```
```

Comment:
I wouldn't put `....` here. Let's put more code to make it clear what it looks like. People can get confused by the rest of it missing.